### PR TITLE
More default inits.

### PR DESCRIPTION
**Pull request by sachaw**
## Thank you for sending in a pull request, here's some tips to get started!

(Please delete all these tips and replace with your text)

- Before starting on some new big chunk of code, it it is optional but highly recommended to open an issue first
  to say "hey, I think this idea X should be implemented and I'm starting work on it. My general plan is Y, any feedback
  is appreciated." This will allow other devs to potentially save you time by not accidentially duplicating work etc...
- Please do not check in files that don't have real changes
- Please do not reformat lines that you didn't have to change the code on
- We recommend using the [Visual Studio Code](https://platformio.org/install/ide?install=vscode) editor and the 'clang-format' extension,
  because automatically follows our indentation rules and it's auto reformatting will not cause spurious changes to lines.
- If your PR fixes a bug, mention "fixes #bugnum" somewhere in your pull request description.
- If your other co-developers have comments on your PR please tweak as needed.

### CODE REVIEW

**Comment by sachaw** [comment]
Protos need merging before merging this.

File: src/main.cpp [hunk 1]
```diff
@@ -157,7 +157,7 @@ void setup()
 #endif
 
 #ifdef DEBUG_PORT
-    if (!config.device.serial_disabled) {
+    if (config.device.serial_enabled) {
```
Comment (thebentern) [inline src/main.cpp:160]: This one may be a problem since I believe it happens before nodedb sets up all of the defaults

Comment (sachaw) [reply src/main.cpp:160]: Can we put in a condition where it's accessed before initialised to handle unset as enabled?

**Comment by sachaw** [comment]
currently hanging on boot:
```
0x400d78cd:0x3ffd5750 0x400e0c95:0x3ffd57a0 0x4012f156:0x3ffd57c0
```

**Pull request merged**